src/common/components/Form/AlertError.tsx

```diff
@@ -0,0 +1,29 @@
+import { AlertCircle } from 'lucide-react';
+
+import { Alert, AlertDescription, AlertTitle } from '../ui/alert';
+
+interface IAlertErrorProps {
+	title: string;
+	message: string;
+	testName: string;
+}
```

Comment:
let's use `Type` instead of `Interface` in these cases. And Rename  to `AlertErrorProps`